client/src/components/ProductShow.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react'
+
+const ProductShow = props => {
+  const [product, setProduct] = useState({
+    id: '',
+    brandName: '',
+    productName: '',
+    description: ''
+  })
```

Comment:
You actually do not need to pre-set these. Unlike in a form, if it's a SHOW page, you can just let them be `undefined` by not specifying them at all. Then, in your `return` below, nothing will show on the page anyway because `product.description`, etc will be `undefined`.